### DISCUSSION TITLE
D3X-Rebirth

### DISCUSSION THREAD

**Discussion by JeodC** [dxx-rebirth/dxx-rebirth#771] [General]
Descent 3 source code is out fyi https://github.com/kevinbentley/Descent3

**Comment by JeodC** [comment]
@vLKp How can I get in touch with you outside of github?

**Comment by vLKp** [comment]
I saw the announcement, I think from someone posting about it on a GOG forum.  While I am not averse to taking Descent 3 into the Rebirth project, that seems premature at this time.  The Rebirth project is meant to keep the games alive, but Descent 3 seems quite lively at the moment.  There are quite a few people actively developing on it elsewhere, and trying to funnel all that into the Rebirth project now would mean either that I'm constantly chasing merges of their work, or pushing people to switch repositories without a good reason.  If the Descent 3 open source project stalled out, I would look at bringing it into Rebirth.  Until then, I'm inclined to stand back and avoid risking a split of the community.

As for outside contact, that came up again in https://github.com/dxx-rebirth/dxx-rebirth/issues/774.  The short answer is that there isn't a good way.  I get maybe one Descent-related e-mail every couple of years.  The Rebirth forum closed years ago.
